pkg/sanity/sanity.go

```diff
@@ -453,6 +453,14 @@ func UniqueString(prefix string) string {
 	return prefix + uniqueSuffix
 }
 
+func UniqueStringWithLength(prefix string, length int) string {
+	str := UniqueString(prefix)
+	if len(str) > length {
+		Panic()
```

Comment:
Do we have to introduce Panic() here?  Can we trim it if it is longer than "length" and make it a best effort to have a unique name?

Reply:
I think we'd better panic here, and make it guaranteed unique. All callers are just using a hard-coded prefix, So if someone do reach here, it should be easy for him to just fix the panic by using a shorter prefix, rather than debugging weird issue caused by non-unique ID.

I mean panic() rather than Panic() here, I'll fix that.